api/src/main/resources/org/openmrs/api/db/hibernate/Provider.hbm.xml

```diff
@@ -1,70 +0,0 @@
-<?xml version="1.0"?>
-<!--
-
-    This Source Code Form is subject to the terms of the Mozilla Public License,
-    v. 2.0. If a copy of the MPL was not distributed with this file, You can
-    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
-    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
-
-    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
-    graphic logo is a trademark of OpenMRS Inc.
-
--->
-<!DOCTYPE hibernate-mapping PUBLIC
-        "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
-        "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd" >
-
-<hibernate-mapping package="org.openmrs">
-
-	<class name="Provider" table="provider">
-
-		<id name="providerId" type="java.lang.Integer" column="provider_id"
-			unsaved-value="0">
-			<generator class="native">
-				<param name="sequence">provider_provider_id_seq</param>
-			</generator>
-		</id>
-
-		<discriminator column="provider_id" insert="false" />
-
```

Comment:
@gitcliff @dkayiwa I did not understand a few things here, why do we need a discriminator column here, since I did not see provider being used as a super class for any other entity class.